### DISCUSSION TITLE
Code Search Update is the worst update in history of GitHub Updates

### DISCUSSION THREAD

**Discussion by CypherpunkSamurai** [community/community#55322] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

**Here's my honest take on GitHub Code Search,**

**I'm sorry but I feel sad to say this update is the worst update I've seen on GitHub, and I mean it from the bottom of my heart.**

I've been using for the past 4-6 years to lookup code, files, proper implementations, and updates of rather old code. "I learnt from other's code", second to stackoverflow GitHub has been a primary source for me to find pieces of code that help my own project. Rather than sitting for someone to reply I'd rather search on GitHub to find something that works.

This update has not only changed that, but it's completely wiped out my ability to research my problems. 

You'd say "oh but there's AI", hear me out, regardless how much AI is trained it commits to being correctly incorrect. Bing still halucinates between similar libraries in different languages giving me functions that don't exist. You can't expect me to depend on it.

I'm not sure if your GitHub's team was having issues with frontend team, or you wiped your code search indexing database, "but this update sucks".

You know this was a mistake when grep finds more code than GitHub code search

**Please revert changes**

**Comment by zztxv** [comment]
same for me

**Comment by mikev339** [comment]
10000000%

Everytime I try to search for anything there's "no results" when before this search update I would have seen results. This update is the worst.

**Comment by mjgruta** [comment]
"This repository's code is being indexed right now. Try again in a few minutes." This has been on for months now. Still cannot search any codes in our private repo. Old search works fine but I cannot use it anymore.

**Reply by lmstearn** [reply]
Got the indexing issue with a repo (not my own), waited a few minutes, and amazingly got the hits. Have they fixed it over your way yet?

**Comment by roschler** [comment]
@CypherpunkSamurai 

GitHub search was previously bad, now it's useless.  I get that same "being indexed" message all the time.

**Comment by ish-xyz** [comment]
it really sucks
